stable/atlantis/README.md

```diff
@@ -39,6 +39,7 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
 | `extraVolumeMounts`                         | List of additional volumes mounted to the container.                                                                                                                                                                                                                                                      | `[]`    |
 | `imagePullSecrets`                          | List of secrets for pulling images from private registries.                                                                                                                                                                                                                                               | `[]`    |
 | `gitconfig`                                 | Contents of a file to be mounted to `~/.gitconfig`.  Use to allow redirection for Terraform modules in private git repositories.                                                                                                                                                                          | n/a     |
+| `gitConfigSecretName`                       | Name of a pre-existing Kubernetes `Secret` containing a `gitconfig` key. Use this instead of `gitconfig` (optional)                                                                                                                                                                           | n/a     |
```

Comment:
if `gitconfig` is all lower case should this also be lowercase as in `gitconfigSecretName`